lib/controllers/spaces.rb

```diff
@@ -21,4 +21,17 @@ class MakersBnB < Sinatra::Base
     redirect '/spaces'
   end
 
+  get '/spaces/view' do
+    @space = Space.first(id: params[:space])
+    erb :'spaces/view'
+  end
+
+  post '/spaces/addDate' do
```

Comment:
PUT http://example.com/widgets/123
Data:
    name = New name
    color = blue

this should be the url for changing the widget with the ID 123. Do we have a page for viewing a specific space yet. 

http://www.andrewhavens.com/posts/20/beginners-guide-to-creating-a-rest-api/

pulled from this guide